### ISSUE TITLE
Take a look at PDF

### ISSUE THREAD

**Issue by michiQ** [yadaguru/yadaguru-app#118]

**Comment by michiQ** [comment]
moving this back in priority because honestly we don't kids will really use.  If it's an easy fix, we'll fix, if not we'll deprecate.

**Comment by michiQ** [comment]
Moving this to backlog.  There's a lot of higher priority stuff in front of it.

**Comment by bobderrico80** [comment]
Yeah, I did a little research on this, and it is probably more effort than we have resources for to get this working consistently on phones. We still have the button in the UI in some contexts, and should probably remove it. I will create a separate issue.
